app/services/workflows/non_passported_workflow.rb

```diff
@@ -68,13 +81,13 @@ def convert_self_employments(self_employments)
       end
 
       def collate_and_assess_gross_income(assessment:, self_employments:, partner_self_employments:)
-        converted_employments = convert_employments(assessment.employments, assessment.submission_date)
+        converted_employments = convert_employments(assessment, assessment.employments, assessment.submission_date)
```

Comment:
What do you think of reducing function arguments by passing assessment object  with additional (type) parameter?
Eg: 
convert_employments(assessment, :employments)
convert_employments(assessment, :partner_employments)

and use assessment.send(type) in definition

Reply:
That's a pattern we have been trying to move away from. We're trying to shrink the code surface (so passing employments to an employments thing rather than the whole assessment)